.github/workflows/test.yml

```
@@ -1184,7 +1184,8 @@ jobs:
             './ukef/cypress/e2e/journeys/portal/deal-cancellation/BSS-EWCS/**/*.spec.js',
             './ukef/cypress/e2e/journeys/portal/deal-cancellation/gef/**/*.spec.js',
             './ukef/cypress/e2e/journeys/portal/amendments/journeys/**/*.spec.js',
-            './ukef/cypress/e2e/journeys/portal/amendments/pages/**/*.spec.js'
+            './ukef/cypress/e2e/journeys/portal/amendments/pages/portal/**/*.spec.js',
+            './ukef/cypress/e2e/journeys/portal/amendments/pages/tfm/**/*.spec.js'
```

@zainUKEF it shouldn't be like this instead ?
```suggestion
            './ukef/cypress/e2e/journeys/tfm/amendments/pages/**/*.spec.js'
```

---

no, there is a tfm folder within portal/amendments/pages/ which deals with the tfm side of portal amendments

---

the reason I put it there is because it is the portal amendment journey.  thoughts @abhi-markan ?